synapse/tests/test_tools_storm.py

```diff
@@ -170,3 +171,35 @@ async def test_tools_storm(self):
                 await s_t_storm.main((lurl, f'!export {path} {{ test:newp }}'), outp=outp)
                 text = str(outp)
                 self.isin('No property named test:newp.', text)
+
+                proc = subprocess.Popen(['python', '-m', 'synapse.tools.storm', lurl], stdout=subprocess.PIPE, shell=True)
+                try:
+                    stdout, _ = proc.communicate(timeout=1)
```

Comment:
This code was removed from another branch.

Reply:
Whups, this was probably me messing up the merge deconfliction :)